db/migrate/20160415200213_add_missing_indexes.rb

```diff
@@ -1,12 +1,11 @@
-class AddMissingIndexes < ActiveRecord::Migration
+class AddMissingIndexes < ActiveRecord::Migration[5.2]
   def change
     add_index :flood_risk_engine_enrollments, :applicant_contact_id
     add_index :flood_risk_engine_enrollments, :organisation_id
     add_index :flood_risk_engine_enrollments, :site_address_id
     add_index :flood_risk_engine_contacts,
               :partnership_organisation_id,
               name: 'fre_contacts_partnership_organisation_id'
-    add_index :flood_risk_engine_organisations, :contact_id
```

Comment:
Why have we removed this?

Reply:
There was a duplicate index error when I ran the migrations. 
The `contact_id` is added as a reference column which should add the index:
https://github.com/DEFRA/flood-risk-engine/blob/c72541ddfe582426ee7a1dfcb2d313b2d324ab8d/db/migrate/20160406132213_create_flood_risk_engine_organisations.rb#L6

Just ran the migrations again locally, and checked in postgres for the index: ` "index_flood_risk_engine_organisations_on_contact_id" btree (contact_id)`,